lib/test.js

```diff
@@ -431,15 +431,15 @@ class Test {
 		this.planError = planError;
 	}
 
-	timeout(ms) {
+	timeout(ms, message) {
```

Comment:
I think there should be some validation that ensures the `message` is a string if it's defined and error out if not. To prevent incorrect usage.

Reply:
Validation and tests added. I now throw errors on incorrect usage just like most of existing validations. I also used similar tests a reference to verify the corresponding behavior.